src/components/Form/StyledForm.js

```diff
@@ -0,0 +1,139 @@
+import dayjs from 'dayjs';
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import StyledButton from '../Button/StyledButton';
+import StyledDiaryHeadline from '../Headline/StyledDiaryHeadline';
+import StyledHeadline2 from '../Headline/StyledHeadline2';
+
+const StyledInput = styled.textarea`
+	display: flex;
+	flex-wrap: wrap;
+	width: 400px;
+	height: 100px;
+	margin: 30px 0;
+	padding: 10px;
+	border: 1px solid var(--rose);
+	border-radius: 5px;
+	color: var(--turq);
+`;
+
+const StyledLabel = styled.label`
+	display: none;
+`;
+
+const StyledUl = styled.ul`
+	display: flex;
+	flex-direction: column-reverse;
+	width: 400px;
+	margin-bottom: 20px;
+	border-radius: 5px;
+	gap: 10px;
+`;
+
+const StyledLi = styled.li`
+	display: flex;
+	flex-direction: column;
+	margin: 0 15px 10px 15px;
+	color: var(--turq_light);
+	gap: 50px;
+`;
+
+const StyledCard = styled.a`
+	display: flex;
+	flex-direction: column-reverse;
+	width: 800px;
+	border: 1px solid var(--turq_light);
+	border-radius: 5px;
+`;
+
+const StyledH4 = styled.h4`
+	display: flex;
+	order: 1;
+	margin-left: 15px;
+	color: var(--turq_light);
+`;
+
+const StyledDate = styled.div`
+	order: 2;
+	margin-left: 15px;
+	color: var(--turq_light);
+`;
+
+function Date({datum}) {
+	return <StyledDate>{dayjs(datum).format('DD.MM.YYYY h:mm A')}</StyledDate>;
+}
+
+export default function StyledForm() {
+	const [entries, setEntries] = useState([]);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+
+		const formData = new FormData(event.target);
+		const data = Object.fromEntries(formData);
+
+		setEntries([...entries, {...data, id: nanoid()}]);
+		event.target.reset();
+	}
+	return (
+		<>
+			<StyledHeadline2 />
+			<form onSubmit={handleSubmit} autoComplete="off">
+				<StyledLabel htmlFor="firstEntry">firstEntry </StyledLabel>
+				<StyledInput
+					type="text"
+					id="firstEntry"
+					name="firstEntry"
+					aria-label="Form for GratitudeDiary"
+					placeholder="Name a thing you are grateful for today"
+					required
+					minLength="3"
+					maxLength="200"
+				/>
+				<StyledLabel htmlFor="secondEntry">secondEntry </StyledLabel>
+				<StyledInput
+					type="text"
+					id="secondEntry"
+					name="secondEntry"
+					aria-label="Form for GratitudeDiary"
+					placeholder="Surely there was another great thing happening today"
+					required
+					minLength="3"
+					maxLength="200"
+				/>
+				<StyledLabel htmlFor="thirdEntry">thirdEntry </StyledLabel>
+				<StyledInput
+					type="text"
+					id="thirdEntry"
+					name="thirdEntry"
+					aria-label="Form for GratitudeDiary"
+					placeholder="Think harder about a last thing that you are grateful for"
+					required
+					minLength="3"
+					maxLength="200"
+				/>
+
+				<StyledButton type="submit">Add to diary</StyledButton>
+			</form>
+			<StyledDiaryHeadline />
+			<StyledUl>
+				{entries.map(entry => {
+					return (
+						<StyledCard key={nanoid}>
```

Comment:
The `key` should be the `id` from your entry:
```suggestion
						<StyledCard key={entry.id}>
```